README.adoc

```diff
@@ -0,0 +1,1763 @@
+= The RSpec Style Guide
+:doctype: article
+
+This RSpec style guide outlines the recommended best practices for real-world
+programmers to write code that can be maintained by other real-world
+programmers.
+
+https://github.com/rubocop-hq/rubocop[RuboCop], a static code analyzer
+(linter) and formatter, has a
+https://github.com/rubocop-hq/rubocop-rspec[`rubocop-rspec`] extension,
+provides a way to enforce the rules outlined in this guide.
+
+This guide assumes you are using RSpec 3 or later.
+
+== How to Read This Guide
+
+The guide is separated into sections based on the different pieces of an entire
+spec file. There was an attempt to omit all obvious information, if anything is
+unclear, feel free to open an issue asking for further clarity.
+
+== A Living Document
+
+Per the comment above, this guide is a work in progress - some rules are simply
+lacking thorough examples, but some things in the RSpec world change week by
+week or month by month. With that said, as the standard changes this guide is
+meant to be able to change with it.
+
+== Table of Contents
+
+* <<layout,Layout>>
+* <<example-group-structure,Example Group Structure>>
+* <<example-structure,Example Structure>>
+* <<naming,Naming>>
+* <<matchers,Matchers>>
+* <<rails,Rails>>
+ ** <<views,Views>>
+ ** <<controllers,Controllers>>
+ ** <<models,Models>>
+ ** <<mailers,Mailers>>
+* <<recommendations,Recommendations>>
+
+== Layout
+
+[[empty-lines-after-describe]]
+&bull; Do not leave empty lines after `feature`, `context` or `describe` descriptions.
+It doesn't make the code more readable and lowers the value of logical chunks.
+^[<<empty-lines-after-describe,link>>]^
+
+[source,ruby]
+----
+# bad
+describe Article do
+
+  describe '#summary' do
+
+    context 'when there is a summary' do
+
+      it 'returns the summary' do
+        # ...
+      end
+    end
+  end
+end
+
+# good
+describe Article do
+  describe '#summary' do
+    context 'when there is a summary' do
+      it 'returns the summary' do
+        # ...
+      end
+    end
+  end
+end
+----
+
+'''
+
+[[empty-lines-between-describes]]
+&bull; Leave one empty line between `feature`, `context` or `describe` blocks.
+Do not leave empty line after the last such block in a group.
+^[<<empty-lines-between-describes,link>>]^
+
+[source,ruby]
```

Comment:
The word "source" is no longer required. You can write this line as `[,ruby]`.

Reply:
It's up to you. I'm just letting you know that "source" is no longer required. This was an improvement that came in Asciidoctor 2.0.0. See https://github.com/asciidoctor/asciidoctor/issues/1117#issuecomment-475418597